### PR TITLE
Update Deprecated features and Summary for March 21 release note

### PR DESCRIPTION
**Pull request by jmbermingham**
https://deploy-preview-1759--axway-open-docs.netlify.app/docs/apim_relnotes/20210330_apimgr_relnotes/

https://deploy-preview-1759--axway-open-docs.netlify.app/docs/apim_relnotes/20210330_apip_relnotes/

* Update **Deprecated** features
* Add a note to the first section, **Summary** (Requested by @blynch-axway)
* In **APIP** release, move deprecation of IE11 from Important changes to Deprecated features.
* Changes for Axway style and improve readability
* Fixed markdownlinter

Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by andreamussap** [comment]
Commit `5f7f0e2 Merge branch 'developmar21' into cms/jmbermingham` was to resolve conflict on list of security vulnerabilities.

File: content/en/docs/apim_relnotes/20210330_apimgr_relnotes.md [hunk 1]
```diff
@@ -114,13 +114,17 @@ For more information, see [Install and configure a metrics database](/docs/apim_
...
+In the [January 2020](/docs/apim_relnotes/20200130_apimgr_relnotes/) update, we announced the deprecation of all the Antivirus filters in API Gateway. This is a reminder that in July 2021 we will remove the Antivirus filters from API Gateway. So, we recommend you to use the API Gateway's ICAP capability, which allows the gateway to integrate with ICAP capable external virus scanners.
 
-### Centos OS ??
+### End of Support notices
 
-placeholder ??
+The following items are end of support (EOS) now:
 
-## Removed features
+* API Gateway and API Manager version [7.7 January 2020 update](/docs/apim_relnotes/20200130_apip_relnotes/).
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20210330_apimgr_relnotes.md:123]: @dginevaAxway if this is true for API Gtw, shouldn't we add the same not in APIP? If yes, I can add it as part of this PR

Comment (jmbermingham) [reply content/en/docs/apim_relnotes/20210330_apimgr_relnotes.md:123]: Yes!

**Review by blynch-axway — APPROVED** [review]
Looks fine to me at this stage with all the various additions etc. Thanks.

**Review by dginevaAxway — COMMENTED** [review]
Approved

**Pull request merged**